config/config.go

```diff
@@ -252,6 +253,7 @@ var defaultConf = Config{
 	Host:             "0.0.0.0",
 	AdvertiseAddress: "",
 	Port:             4000,
+	Cors:             false,
```

Comment:
@xuechunL  How do you want to use those api~?

default disable `cors` and enable when you app deploy with tidb is ok~?